### DISCUSSION TITLE
Free trail isnt working

### DISCUSSION THREAD

**Discussion by PurHur** [community/community#19551] [Copilot Conversations] [answered]
I cant use the free trail it says that i have no access.

I tried to use it without payment enabled

**Comment by D7EAD** [comment]
Hi!

Have you followed [this](https://docs.github.com/en/billing/managing-billing-for-github-copilot/managing-your-github-copilot-subscription) guide for setting up a free 60-day trial of Github Copilot? If so, and you continue to have issues, please raise a ticket with Github's Support Engineers to resolve this issue. You can do so, [here](https://support.github.com/contact/account).

If I've managed to help you, please consider **Marking as Answer** so others can more easily find the solution to this issue if they so encounter it.

**Comment by davecheney** [comment, marked as answer]
@PurHur there is a problem with your billing details; please contact support and ask them to reset your billing status.
